packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1015,7 +1004,8 @@ function beginWork(
         );
         break;
       case ContextProvider:
-        pushProvider(workInProgress, 0);
+        const newValue = workInProgress.memoizedProps.value;
```

Comment:
I think it's probably best practice to wrap this in a block because this variable name is in scope of the whole function. If you need the same name in a different branch.

Reply:
Switch case trap